WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -107,14 +116,18 @@ private extension SupportTableViewController {
                                 NavigationItemRow.self,
                                 TextRow.self,
                                 HelpRow.self,
+                                DestructiveButtonRow.self,
                                 SupportEmailRow.self],
                                tableView: tableView)
         tableHandler = ImmuTableViewHandler(takeOver: self)
         reloadViewModel()
         WPStyleGuide.configureColors(view: view, tableView: tableView)
-        // remove empty cells
-        tableView.tableFooterView = UIView()
-
+        tableView.tableFooterView = UIView() // remove empty cells
+        if let headerConfig = configuration.meHeaderConfiguration {
+            let headerView = MeHeaderView()
```

Comment:
Ideally, the `MeHeaderView` should be initialized with the `config` param, but the `MeHeaderView` is used elsewhere and would require some refactoring to be done.